### PR TITLE
[ICP-6819] GoControl - watts update automatically every 15 mins. ICP-6819 fix.

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @PKacprowiczS @MMateusiakS @KKlimczukS 
Could you review this?

### CODE REVIEW

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy [hunk 1]
```diff
@@ -94,16 +94,16 @@ metadata {
 def installed() {
 	log.debug "installed()"
 	// Device-Watch simply pings if no device events received for 32min(checkInterval)
-	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
-	if (zwaveInfo?.mfr?.equals("0063")) { // These old GE devices have to be polled
+    initialize()
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: nit: the indentation here looks off

**Review by tpmanley — DISMISSED** [review]

**Comment by greens** [comment]
Yes. Please fix the indentation.

**Comment by greens** [comment]
In the Jira ticket, Shravan quoted this from the mfr:

>Parameter 13 is the time between unsolicited frames for the kWh accumulated meter reports.

>This is configurable from 1 minute to 255 minutes

Can you try adjusting this parameter to fix this issue before we implement polling?

**Comment by MGoralczykS** [comment]
I fix spaces and also set parameter 13 to 15 mins. 
@greens Could you review this?

**Comment by greens** [comment]
Did setting the configuration parameter actually make it report every 15 minutes? If the configuration parameter works I'd prefer to leave this device off of the forced polling.

**Comment by MGoralczykS** [comment]
Paremeters only reports kWh every 15 minutes. For watts update we need make this schedule.

**Review by greens — APPROVED** [review]

**Pull request merged**